src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -62,18 +63,18 @@
 						</ActionButton>
 					</Actions>
 				</div>
-				<div>
+				<div class="new-message-form__emoji-picker">
 					<EmojiPicker
 						container="#content-vue"
 						@select="addEmoji">
 						<button
 							type="button"
 							:disabled="disabled"
-							class="nc-button nc-button__main"
+							class="nc-button nc-button__main emoji-picker-button"
 							:aria-label="t('spreed', 'Add emoji')"
 							:aria-haspopup="true">
 							<EmoticonOutline
-								:size="16"
+								:size="18"
```

Comment:
I thought we discussed in the design call to stay with 16 for now, to not introduce too many different sizes?

Reply:
This type of button hasn't been discussed yet, it has no background even on hover now that it's in the new message form and it looks really small at 16px.

Reply:
The change doesn’t seem to be related to the pull request, and since elsewhere the size is 16, we should keep it like that? (E.g. in other components like the sidebar tab icons.)